modules/ROOT/partials/configuration/ai_shortcuts.adoc

```diff
@@ -0,0 +1,107 @@
+[[ai_shortcuts]]
+== `ai_shortcuts`
+
+The `+ai_shortcuts+` option controls the list of {pluginname} shortcuts available in the `+AI Shortcuts+` toolbar button and menu item.
+
+This option can be configured with an array to present a customised set of {pluginname} shortcuts.
+
+As well, it can be set to a `+Boolean+` value to control the use of the default list of {pluginname} shortcuts.
+
+When not specified, or set to `+true+`, the {pluginname} shortcuts toolbar button and menu item present and display the default set of shortcuts included with the {pluginname}.
+
+When set to `+[]+` (an empty array), or `+false+`, the {pluginname} shortcuts toolbar button and menu item do not present in the {productname} instance.
+
+When configured with an instance-specific object array, the {pluginname} shortcuts toolbar button and menu item present, and display the configured shortcuts when activated.
+
+*Type:* `+Array+` of `+Objects+`, or `+Boolean+`
+
+*Default value:* 
+[source, js]
+----
+[
+  { title: 'Summarize', prompt: 'Briefly summarize this.' },
+  { title: 'Explain', prompt: 'Give a short explanation of this.' },
+  { title: 'Improve grammar', prompt: 'Correct grammar.' },
+  { title: 'Improve writing', prompt: 'Enhance this text.' },
+  { title: 'Simplify language', prompt: 'Simplify this.' },
+  { title: 'Make longer', prompt: 'Expand this.' },
+  { title: 'Make shorter', prompt: 'Condense this.' },
+  { title: 'Change tone', subprompts: [
+    { title: 'Descriptive', prompt: 'Make this descriptive.' },
+    { title: 'Formal', prompt: 'Make this formal.' },
+    { title: 'Casual', prompt: 'Make this casual.' },
+    { title: 'Humour', prompt: 'Add humour to this.' },
+    { title: 'Inspirational', prompt: 'Inspire reader with this.' },
+    { title: 'Persuasive', prompt: 'Make this persuasive.' },
+  ] },
+  { title: 'Change style', subprompts: [
+    { title: 'Business', prompt: 'Make this business-like.' },
+    { title: 'Legal', prompt: 'Legal style this.' },
+    { title: 'Journalism', prompt: 'Journalistic style this.' },
+    { title: 'Medical', prompt: 'Medical style this.' },
+    { title: 'Poetic', prompt: 'Make this poetic.' },
+  ] },
+]
+----
+
+NOTE: The default shortcuts are subject to change in an upcoming release. If you intend to keep these shortcuts, include them within your integration.
+
+
+=== Example: using `ai_shortcuts` to present a customised set of {pluginname} shortcuts
```

Comment:
@tiny-ben-tran @MitchCTiny @brian-forte 

How about incorporating @tiny-ben-tran changes as well as replace code snippet with below code snippet. I'm proposing the change so that users can try overriding shortcuts just by copy and pasting the snippet into fiddle. 


Please review.

`tinymce.init({
  selector: 'textarea',  // change this value according to your html
  plugins: 'ai',
  toolbar: 'aidialog aishortcuts',
    ai_request: (request, respondWith) => respondWith.string(() => Promise.reject("See docs to implement AI Assistant")),
  ai_shortcuts: [
    { title: 'Screenplay', prompt: 'Convert this to screenplay format.' },
    { title: 'Stage play', prompt: 'Convert this to stage play format.' },
    { title: 'Classical', subprompts:
      [
      { title: 'Dialogue', prompt: 'Convert this to a Socratic dialogue.' },
      { title: 'Homeric', prompt: 'Convert this to a Classical Epic.' },
      ]
    },
    { title: 'Celtic', subprompts:
      [
      { title: 'Bardic', prompt: 'Convert this to Bardic verse.' },
      { title: 'Filí', prompt: 'Convert this to Filí-an verse.' },
      ]
    },
  ]
});`

Reply:
[Edited code-sample added](https://github.com/tinymce/tinymce-docs/commit/2e1a3c2f28d585b2f3ec5d166b001eb54e3a43ae).